Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -1355,7 +1355,7 @@ final class AppDelegateViewModelTests: TestCase {
     )
   }
 
-  func testVisitorCookies() {
+  func testVisitorCookies_ApplicationDidFinishLaunching() {
```

Comment:
This test is unchanged just to prove that existing functionality wasn't affected.